Eradication/MigrationInfoVector.cpp

```diff
@@ -91,14 +458,38 @@ namespace Kernel
         // --- We only want to save raw migration rates for female 
         // --- vectors, because male vector rates do not get modified
         // ----------------------------------------------------------
-        if( gender == Gender::FEMALE )
+
+       
+        // Using raw migration rates to calculate fraction traveling instead of 
+        // back-calculating from the normalized r_cdf and dealing with float issues
+        // caused by last r_cdf parameter hard-coded to 1 (to avoid float point issues)
+
+        std::vector<float> fraction_traveling;
+        if( m_TotalRate > 0 )
```

Comment:
this is where I'd have to total the rate

Reply:
it's double convenient as we can fill the m_TotalRate<>ByIndex with the total rate here

Reply:
outdated